src/main/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/entity/DecisionAndActions.kt

```diff
@@ -0,0 +1,57 @@
+package uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.MapsId
+import jakarta.persistence.OneToOne
+import jakarta.persistence.Table
+import java.time.LocalDate
+
+@Entity
+@Table
+data class DecisionAndActions(
+  @Id @MapsId("record_id") @OneToOne(fetch = FetchType.LAZY) @JoinColumn(
+    name = "record_id",
+    referencedColumnName = "record_id",
+  ) val referral: Referral,
+
+  @OneToOne @JoinColumn(
```

Comment:
I think this and the ref data one below are many to ones?

Reply:
You're right, fixed